### PR TITLE
initial setup + training routes

### PR DESCRIPTION
**Pull request by luciipak**

### CODE REVIEW

File: deployments/pheno/services/apiv1/src/endpoints/trainings.py [hunk 1]
```diff
@@ -0,0 +1,815 @@
...
+            detail=f"Error uploading training images to s3: {e}",
+        )
+
+
+@router.post(
+    "/submitNewTraining/{user_id}",
+    response_description="redirects to step 2/3 and passes certain params to next view",
+    description="Step 1/3 when starting a new training. Route creates new training entry in db with user-inputted data then uploads to s3.",
+)
+async def new_train_submit(
```
Comment (luciipak) [inline deployments/pheno/services/apiv1/src/endpoints/trainings.py:292]: haven't been able to figure out how to use form data with a Pydantic model to clean this up. I can just send this in the request body if I need to and store it as an object in the FE instead.

File: deployments/pheno/services/apiv1/src/endpoints/trainings.py [hunk 2]
```diff
@@ -0,0 +1,815 @@
...
+            # update review status on user type
+            skip_review = (
+                "yes"
+                if account["type"]
+                in ["admin", "mo", "tenaya", "fountain", "shadi", "juan", "charlene", "chronus"]
+                else "no"
+            )
+
+            # insert new training data
+            await cur.execute(
```
Comment (luciipak) [inline deployments/pheno/services/apiv1/src/endpoints/trainings.py:352]: I was wondering if there was any other way of performing this query?

**Comment by luciipak** [comment]
I haven't added the use of the AsyncPoolDep class for the db connection yet, but will incorporate that when I setup the rest of the db credentials etc. I think that should be the next task I set up when I get back to Pheno

**Review by jperson — APPROVED** [review]

File: deployments/pheno/services/apiv1/src/endpoints/trainings.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (luciipak) [reply deployments/pheno/services/apiv1/src/endpoints/trainings.py:181]: For some reason it was showing an old commit where i hadn't done that yet.. but let me know if you see those changes

[the remark this answers is not in the document]
Comment (luciipak) [reply deployments/pheno/services/apiv1/src/endpoints/trainings.py:181]: I had also removed all the exceptions and added a class to catch them under utils. I added it here:
`router = APIRouter(prefix="/train", tags=["trainings"], route_class=RouteErrorHandler)`

**Review by jperson — APPROVED** [review]

**Pull request merged**